apps/examples/wc-vanilla/package.json

```diff
@@ -3,16 +3,20 @@
   "version": "0.2.10",
   "private": true,
   "type": "module",
+  "installConfig": {
+    "hoistingLimits": "workspaces"
```

Comment:
Not to do with this change - but this reminded me - do we need volta config for each example app, even if it just extends the root config?

Reply:
Being absolutely honest, I've no idea 😆  Might be worth asking in the engineering channel and we can raise a separate ticket to add it (or just add it quickly as a separate PR)

Reply:
@xander-marjoram I'm resolving this as it shouldn't block this PR but please do pick this up with the team 👍